README.md

```diff
@@ -1,118 +1,47 @@
-# Template repository for a Next.js application
+# Template repository for application infrastructure
 
-This is a template repository for a Next.js-based application. While this template should be opinionated in some ways in order to reduce setup overhead where possible, it should be application-agnostic, meaning that any type of Next.js application should be able to be created from this template.
+## Overview
 
-## Contents
+This is a template repository for a to set up foundational infrastructure for your application in AWS. This template includes setup for:
 
-This template includes setup for:
+* **Team workflows** - PR template, architecture decision record (ADR) template, Makefile.
+* **Infrastructure as code** - infrastructure for terraform backends, including S3 buckets and DynamoDB table for locking terraform state
```

Comment:
small wording note — based on the general title I would expect this to include app infrastructure. Maybe a better term would be something like "Infrastructure bootstrap" or "foundational infrastructure"? 🤷 just thinking out loud.

Reply:
what about "account-level foundational infrastructure"?